console/my-account/concepts.mdx

```diff
@@ -117,22 +93,6 @@ MFA is an authentication method which allows you to easily secure your Scaleway
 
 <Concept>
 
-## Organization
-
-Organizations is Scaleway's Multi-User feature. Every Scaleway account has an associated Organization. An Organization is a resource system, with hierarchically organized accesses and permissions. An Organization can have one or more associated [projects](#project): the Organization is the root account of each of its Projects. Organizations can have one or more [members](#members) with different [roles](#roles). Organizations allow you to centrally manage and share resources across multiple accounts depending on the permissions granted by the [owner](#owner).
-
-</Concept>
-
-<Concept>
-
-## Organization Dashboard
-
-The Organization Dashboard is the display you see when you log in to [console.scaleway.com](https://console.scaleway.com/). On this dashboard you can see the name of your [Organization](#organization), and depending on your [role](#roles), an overview of your resources and a breakdown of your resource consumption and billing. As you add and use resources, the Organization Dashboard stays up to date.
-
-</Concept>
-
-<Concept>
-
 ## Organization Quotas
 
 Every [Organization](#organization) has quotas, which are limits on the number of Scaleway resources they can use. Quotas are per product (eg Instances) and product type (eg GP1-L instance). Quotas are designed to prevent abuse, and can be viewed on the [Organization Account page](https://console.scaleway.com/account/organization/profile) of the console.
```

Comment:
```suggestion
Every [Organization](#organization) has quotas, which are limits on the number of Scaleway resources they can use. Quotas are per product (eg Instances) and product type (eg GP1-L instance). Quotas are designed to prevent abuse, and can be viewed on the [Organization Account page](https://console.scaleway.com/account/organization/profile) of the console. Find out more about quotas with our [dedicated documentation](/console/my-account/reference-content/account-quotas/).
```

Reply:
Note: this concept is also replicated on the My Project concepts page...maybe we don't want it in both places?

Reply:
DELETE ORGANIZATION QUOTAS CONCEPT FROM MY PROJECT